tests/core/test_model.py

```diff
@@ -562,6 +563,62 @@ def value_max(cls, df: pd.DataFrame) -> Iterable[bool]:
         schema.validate(df, lazy=True)
 
 
+def test_registered_dataframe_checks(
+    extra_registered_checks,
```

Comment:
`extra_registered_checks` looks like an artifact from a previous iteration. Can we remove it?

Reply:
No, that's actually I fixture I added. It mocks the `_CheckBase.REGISTERED_CUSTOM_CHECKS` dictionary and adds a check. I use it to revert any extension checks that I add in the test.